### ISSUE TITLE
Chrome Tab Crash on Selfie Segmentation

### ISSUE THREAD

**Issue by mauriciogior** [google/mediapipe#2631]
**System information** (Please provide as much relevant information as possible)

- Custom code using Selfie Segmentation.
- Using Chrome 94 on MacOS Big Sur Apple Silicon.

**Describe the current behavior:**
Right on the first `selfieSegmentation.send` run, the tab crashes. After more tests, it feels like it crashes if I access `enumerateDevices` at some point

**Other info / Complete Logs :**
[8b2b4b70-1764-405f-a0fa-92a1203da208.dmp.zip](https://github.com/google/mediapipe/files/7282437/8b2b4b70-1764-405f-a0fa-92a1203da208.dmp.zip)

**Comment by sgowroji** [comment]
Hi @mauriciogior, Could you please share the logs in the txt file or trace file to investigate the issue. Thanks!

**Comment by mauriciogior** [comment]
Hi @sgowroji, I don't see anything on Developer Tools since when it crashes it disconnects from the tab. I attached the dump Chrome generated. Could you give me guidance to get the trace file?

Thanks.

**Comment by tyrmullen** [comment]
@mauriciogior I noticed this is happening with custom code using the selfie-segmentation API.  Does the selfie-segmentation MediaPipe CodePen have the same problem for you?

**Comment by mauriciogior** [comment]
Hi @tyrmullen, yes, it is custom code and the CodePen example works (I also tried the one you have in your docs and it also works). What I'm trying to figure out is why the whole tab crashes and how can I fix it -- I can't see any RuntimeExceptions since the Developer Tools disconnects from the tab when it crashes.

My custom code does have more complexity and uses quite a bit of ram, but the video part is basically a copy & paste from the example. I'm also sending this video through WebRTC, but it crashes before even getting into that stage.

**Comment by tyrmullen** [comment]
Glad to hear it-- it didn't sound like anything I had seen before on the MediaPipe side.  Closing as resolved!

If something like this pops up again (and for most connection issues in general), the "Network" tab of the Chrome Developer Console can hopefully shed some light on which resources are having trouble being successfully downloaded (and can disable browser caching and simulate spotty network conditions); I've found it to be invaluable for that sort of thing.
